layouts/index.html

```diff
@@ -169,6 +169,12 @@ <h2 class="title--answer">Mostly...</h2>
         <a href="https://qutebrowser.org/">qutebrowser</a>
         </small>
       </li>
+	<li class="list__item--ok">
+        Login Manager:
+	<a href="https://sr.ht/~kennylevinsen/greetd/">Greetd</a>,
+	<a href="https://wiki.gnome.org/Projects/GDM">GDM</a>,
+	<a href="https://github.com/max-moser/lightdm-elephant-greeter">Elephant-greeter</a>
+      </li>
```

Comment:
A few things to change:

- Use spaces and not tabs for indentation
- Indent <a>... 1 level more
- Sort items alphabetically, by project name
- "Login Manager" -> "Login manager"
- Put that section after "Image viewer"

Reply:
Cool will do. 👍🏻